addons/mail/tests/test_mail_channel_partner.py

```diff
@@ -222,3 +222,9 @@ def test_channel_public(self):
             self.public_channel.with_user(self.user_portal).add_members(self.user_portal.partner_id.ids)
         with self.assertRaises(AccessError):
             self.public_channel.with_user(self.user_public).add_members(self.user_public.partner_id.ids)
+
+    def test_channel_partner_invite_with_guest(self):
+        guest = self.env['mail.guest'].create({'name': 'Guest'})
+        self.public_channel.add_members(guest_ids=[guest.id])
+        search = self.env['res.partner'].search_for_channel_invite("", channel_id=self.public_channel.id)
+        self.assertEqual(len(search.get('partners')), 5)
```

Comment:
This test is a bit weak in regards to number of users in demo data.

It would be better to create a user with a specific name, and use that name as search term.

Reply:
note: we usually use `[ ]` rather than `get` to access dict element when we know it is set / don't need the default.

Reply:
Done